UniMath/Books/README.md

```diff
@@ -0,0 +1,24 @@
+# Books
+
+UniMath is a vast library. This package is a navigational and organizational aid,
+its aim is to establish correspondences between the results in UniMath and those
+presented in some familiar textbooks.
+
+The names serve as a guide only, they are not intended to be referenced in other
```

Comment:
If the names are not intended to be referenced, then why have them?

Reply:
Maybe they should all be local.  (Some are, some are not.)

Reply:
Yes, I think they should all be local. They are intended to be indicative of where the result is from in the text (see the examples). An alternative would be to have them in comments, but then we don't get automatic checking for duplicate entries as we do with local definitions.

Reply:
More generally, the point of having these listed even though they aren't to be used directly in UniMath is to have them as a navigational aid: a third option besides `Search` and grepping through files for finding the names of results you need.

Reply:
Perhaps there is a way to put the reference to "Definition 1.1.9" in a comment instead of in an identifier.  To do that, we'd have to find another way to ensure that @CategoryTheory.Core.Isos.iso exists...  I don't know.

One problem is that nothing ensures the definition of @CategoryTheory.Core.Isos.iso doesn't changed later on, so that it's no longer identical to the thing in the book.

Reply:
@DanGrayson That's true, I hadn't thought of that. However, since most of these books are based in set theory, the comparison is already somewhat of a metaphor. I'd be quite surprised if the definition changed to something conceptually unrelated.